app/services/trainees/create_from_dttp.rb

```diff
@@ -6,6 +6,9 @@ class CreateFromDttp
     include HasDttpMapping
 
     INVALID_TRN = "999999999"
+
+    TRN_REGEX = /^([0-9]{7,})$/.freeze
```

Comment:
```suggestion
    TRN_REGEX = /^\d{7}$/.freeze
```
https://www.gov.uk/guidance/teacher-reference-number-trn seems to suggest that a TRN is always a 7 digit number.
if that is definitely what we expect, this should also take care of the `999999999`, and we wouldn't need a special handling for it.

Reply:
thanks Arlston - after doing some research on this, looking at existing `dfe_trn` variations from DTTP and speaking to Rob Collier who works with DQT, we decided to accept TRNs that are 6 or 7 digits long - historically, it is possible TRNs with less than 7 digits might have dropped leading/trailing 0s. So we'll save those ones too in case they are valid.

Reply:
good idea, makes sense about historical values